include/dxc/Support/HLSLOptions.td

```diff
@@ -330,7 +330,12 @@ def Fc : JoinedOrSeparate<["-", "/"], "Fc">, MetaVarName<"<file>">, HelpText<"Ou
 //def Fx : JoinedOrSeparate<["-", "/"], "Fx">, MetaVarName<"<file>">, HelpText<"Output assembly code and hex listing file">;
 def Fh : JoinedOrSeparate<["-", "/"], "Fh">, MetaVarName<"<file>">, HelpText<"Output header file containing object code">, Flags<[DriverOption]>, Group<hlslcomp_Group>;
 def Fe : JoinedOrSeparate<["-", "/"], "Fe">, MetaVarName<"<file>">, HelpText<"Output warnings and errors to the given file">, Flags<[DriverOption]>, Group<hlslcomp_Group>;
-def Fd : JoinedOrSeparate<["-", "/"], "Fd">, MetaVarName<"<file>">, HelpText<"Write debug information to the given file or directory; trail \\ to auto-generate and imply Qstrip_priv">, Flags<[CoreOption, DriverOption]>, Group<hlslcomp_Group>;
+def Fd : JoinedOrSeparate<["-", "/"], "Fd">, MetaVarName<"<file>">,
+  HelpText<"Write debug information to the given file, or automatically named file in directory when ending in '\\'; implies Zi and Qstrip_debug">,
+  Flags<[CoreOption, DriverOption]>, Group<hlslcomp_Group>;
+def pdb_alt_path : JoinedOrSeparate<["-", "/"], "pdb-alt-path">,
```

Comment:
Would it be clearer to name this pdb-embed-path?

Reply:
We could change the name, but I chose this since it was the same as the linker option name (with style change for consistency - linker option is /PDBALTPATH).  I’d prefer we just had a -pdb-name or -embed-pdb-name, making it clear that your embedding the pdb name, and not embedding a path in the pdb, and ‘name’ because that’s what we have been using everywhere else...   This sort of sets up a naming conflict, where we should probably choose which way to go and make it consistent.

Reply:
Oh, I didn't know about link.exe's  `/PDBALTPATH`. Let's keep it this way then!